### PR TITLE
Add Nullish Coalescing Operator

### PR DESCRIPTION
**Pull request by mrna0**
I saw that there is no nullish operator here.
I don't know if I thought it right, but maybe it is.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2096,11 +2096,13 @@ Other Style Guides
     const foo = a ? a : b;
     const bar = c ? true : false;
     const baz = c ? false : true;
+    const qux = a ? a : b;
 
     // good
     const foo = a || b;
     const bar = !!c;
     const baz = !c;
+    const qux = a ?? b;
```
Comment (ljharb) [inline README.md:2105]: actually `a ? a : b` is equivalent to `a || b`.

The equivalent to `a ?? b` is `a != null ? a : b`.

**Review by ljharb — CHANGES_REQUESTED** [review]
Until babel-preset-airbnb supports nullish coalescing (which it already might), this guide shouldn't. Can you confirm?

File: README.md (hunk 1 above)
Comment (mrna0) [reply README.md]: I changed it. Thank you.

**Comment by mrna0** [comment]
> Until babel-preset-airbnb supports nullish coalescing (which it already might), this guide shouldn't. Can you confirm?

Yes I confirm.

**Comment by mrna0** [comment]
I missed the semicolon.
I am sorry for this.

**Pull request merged**